.github/workflows/staging-build-windows.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     - name: Build Kibana
       env:
         install4j_license: ${{ secrets.INSTALL4J_LICENSE }}
-      run: ./kibana/windows/opendistro-windows-kibana-build.sh
+      run: ./kibana/windows/opendistro-windows-kibana-build.sh windows x64
```

Comment:
This default value. "windows x64"

Reply:
Same as other PR we better specify it instead of relying on default value, as default is just a backup.

Reply:
Ok.